Enterprise-CMCS/macpro-mdct-qmr
# MDCT-2765 Pt. I: Add New Data Source Type Validation

### Description

Adding a new validation for data source selection to FFY2023 measures

### Related ticket(s)


MDCT-2765

---
### How to test

- Log in as state user
- Navigate to FY2023
- Select a `Data Source` but don't select anything else, like so:

![Screenshot 2023-07-27 at 5 28 13 PM](https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/99458559/3cc28d28-9790-482d-ae31-45c7c23d289b)

Validate the measure and you should see this error message:

![Screenshot 2023-07-27 at 5 28 23 PM](https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/99458559/8ae8d601-19b1-4ad8-8b92-2724a16740bf)

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
